zebra/redistribute.c

```diff
@@ -730,8 +730,12 @@ void zebra_import_table_rm_update()
 				continue;
 
 			rmap_name = zebra_get_import_table_route_map(afi, i);
-			if (!rmap_name)
-				return;
+			if ((!rmap_name) || (strcmp(rmap_name, rmap) != 0)) {
+				if (IS_ZEBRA_DEBUG_EVENT)
+					zlog_debug("%s : rmap %s not matching",
```

Comment:
When we have 100's to thousands of route-maps, do we really want to display this debug?  I do not think we do.